lib/perl/Genome/WorkflowBuilder/Command.pm

```diff
@@ -154,6 +154,12 @@ sub _get_ptero_lsf_parameters {
     $set_lsf_option->('queue', $attributes{lsfQueue});
     $set_lsf_option->('projectName', $attributes{lsfProject});
 
+    my $default_job_group = join('/',
```

Comment:
This isn't really the 'default' since it cannot be overwritten.  Perhaps the config could default to the empty string (or some sentinel value) and if it isn't overwritten then this logic could be invoked (using /genome/<username>).